telco-hub/configuration/reference-crs/required/acm/pre-requeriments.md

```diff
@@ -20,6 +20,26 @@ stringData:
       insecure: true
       access_key: "F37GAMG8...........6NU"
       secret_key: "OylYlj5Y...........iFtTcwoBPF9EdEqSMUF0"
+```
+If the S3 endpoint uses the HTTPS (443) port the following config must be used to allow unknown CAs:
```

Comment:
In the reference we are using internal ODF to provide storage. In that case the Secret is not needed, correct?